convention.md

```diff
@@ -295,7 +299,6 @@ homie/kitchen-light/light/power/set ← "true"
 ```
 
 The device would then turn on the light, and update its `power` state.
-This provides pessimistic feedback, which is important for home automation.
```

Comment:
I would keep "pessimistic feedback" in the document.

Reply:
I have changed the wording, maybe you can come up with a different approach. I'd rather not have the original sentence, the statement is made without any proof.

Reply:
How about:

> After the device successfully received and processed the command, pessimistic feedback is given by publishing the new state of the property:

Reply:
As soon as feedback is given (publishing happened), it is neither pessimistic nor optimistic feedback, it is just feedback. The process/procedure itself (publish to prop/set -> processing -> publish to prop) is pessimistic feedback and that isn't conveyed in this sentence.

Reply:
Completely correct. Okay let's leave it out. I'll edit my other comment.